### DISCUSSION TITLE
Nodebb deploys fails

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#284] [Installation] [answered]
Hi Team,

CSP: GCP
I am trying to deploy a report it fails
Build version: release-4.6.0_RC3
Nodebb: v1.18.6

Pod errors: CrashLoopBackOff

`ReplyError: MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.`

**Comment by beepdot** [comment, marked as answer]
Redis is not able to save snapshots to disk. Please check the redis logs. Most likely its due to permission issue. We have already solved a similar issue in another thread. Can you please search for the issues first before posting new issues?

**Reply by tsprasath** [reply]
@keshavprasadms  I have resolved the issue. Please close the thread.
